### PR TITLE
imgProc/sizeDist bug fixes and additions -- Addition of new example wrappers

### PR DESCRIPTION
**Pull request by stechma2**
Modified imgProc/sizeDist to allow for a small number of PECAN/other case-specific processing.

Added a new version of the python automatic processing wrapper which now takes command line arguments to eliminate the need for unique python wrappers _and_ slurm batch scripts.

Added a few additional matlab wrapper script examples (specific to PECAN, but extensible to other projects). Similar in many ways to existing wrappers, so feel free to drop these from the merge if you'd like - just wanted to share if they'd be helpful.

### CODE REVIEW

**Comment by stechma2** [comment]
Also - don't be alarmed by the number of additions/changes to sizeDist... These are largely due to formatting changes (tabs, etc...)

File: imgProc_sm.m [hunk 1]
```diff
@@ -155,9 +155,17 @@ function imgProc_sm(infile, outfile, probename, n, nEvery, projectname, varargin
 warning off all
 diode_stats = zeros(1,diodenum);
 
-if strcmp(projectname, 'PECAN')  % For example for PECAN dataset 
-    disp('Testing...')  %% Add project specific code if you like
-end
+%if strcmp(projectname, 'PECAN')  % For example for PECAN dataset 
+%    if strcmp(probename, 'PIP')
+%        TimeOffsetSec = -12*3600.0;
```
Comment (joefinlon) [inline imgProc_sm.m]: Were these commented lines in use when you were testing different correction techniques? I wasn't sure if they should still remain in the submitted code.

File: imgProc_sm.m [hunk 2]
```diff
@@ -553,6 +562,30 @@ function imgProc_sm(infile, outfile, probename, n, nEvery, projectname, varargin
...
+                    if (exist('TimeOffsetSec') && TimeOffsetSec ~= 0 && (handles.month ~= 7 && handles.day ~= 1))
+                    	if (firstTime == 1 && kk == 1)
+                    		fprintf('Time correction applied to %s: %d seconds\n',probename,TimeOffsetSec);
+                    	end
+						%fprintf('Time in seconds before correction: %d\n',time_in_seconds(kk));
+						%fprintf('Particle hour before correction: %d\n',part_hour(kk));
+						%fprintf('Particle minute before correction: %d\n',part_min(kk));
+						%fprintf('Particle sec before correction: %d\n',part_sec(kk));
+						
+						time_in_seconds(kk) = time_in_seconds(kk) + TimeOffsetSec;
```
Comment (joefinlon) [inline imgProc_sm.m]: I can test these corrections on GCPEx CIP data when I get around to it next week, and see if these need to be implemented for CIP/PIP data for any project type.

File: imgProc_sm.m (hunk 1 above)
Comment (stechma2) [inline imgProc_sm.m]: I originally had the `TimeOffsetSec` variable fed in by the `runSizeDist_example.m` script, but opted to streamline things in this branch for the sake of universalness. However, upon testing it there were a number of bugs that still need working out. You can feel free to remove this from the pull request for now.

File: imgProc_sm.m (hunk 2 above)
Comment (stechma2) [inline imgProc_sm.m]: Thanks Joe. I'll let you know if I make any progress in testing with the PECAN data as well.

**Comment by stechma2** [comment]
I've made some changes to the commits in the original pull request along with some new additions. Namely, netCDF variable types have been modified in `imgProc_sm.m` to improve computational efficiency, and to reduce output file sizes. Additionally, some additions specific to DMT probes were made to `imgProc_sm.m` which handle corrupt image records as seen during the PECAN project.

**Comment by joefinlon** [comment]
Thanks for your hard work with this! Looking forward to smaller PBP file sizes as well.

**Pull request merged**